src/io/util.cpp

```diff
@@ -0,0 +1,24 @@
+#include "aikido/io/util.hpp"
+
+namespace aikido {
+namespace io {
+
+const SkeletonPtr makeBodyFromURDF(
+    const std::shared_ptr<aikido::io::CatkinResourceRetriever>
+        resourceRetriever,
+    const std::string& uri,
+    const Eigen::Isometry3d& transform)
+{
+  dart::utils::DartLoader urdfLoader;
+  const SkeletonPtr skeleton = urdfLoader.parseSkeleton(uri, resourceRetriever);
+
+  if (!skeleton)
+    throw std::runtime_error("unable to load '" + uri + "'");
+
+  dynamic_cast<dart::dynamics::FreeJoint*>(skeleton->getJoint(0))
```

Comment:
I think we can also make this `skeleton->getRootJoint()` instead of `skeleton->getJoint(0)`. @jslee02 can you confirm?

Reply:
Correct!

Also, we should check whether `skeleton` is not empty.